api/data_explorer/__main__.py

```diff
@@ -14,6 +14,7 @@
 from elasticsearch_dsl import HistogramFacet
```

Comment:
I think this is fine as a patch for some of the data we currently have, however it won't address numbers much larger than 100, and won't take into account the distribution of the data.